inductiva/_benchmarks/methods.py

```diff
@@ -58,21 +59,25 @@ def _analyze_task(task: Task, report: Dict[str, Any], skipped_tasks: List[Task],
             "machine_count": resource.num_machines,
             "tasks": []
         }
+    metadata_filename = "input.json"
+    inputs_path = task.download_inputs(filenames=[metadata_filename])
+    metadata_path = inputs_path.joinpath(metadata_filename)
+    with open(metadata_path, mode="r", encoding="utf-8") as metadata_file:
+        metadata_content = json.load(metadata_file)
```

Comment:
This works but I don't like this approach. I'm fine with having the method download_inputs, but we should not be using it for this. We should call an api endpoint to get this metadata for that task.

Reply:
Sure, we could implement a method in the class 'Task'. but I think it's better to leave that for a separate PR.